bootstrap/peer.go

```diff
@@ -16,6 +16,7 @@ import (
 
 	"storj.io/storj/bootstrap/bootstrapweb"
 	"storj.io/storj/bootstrap/bootstrapweb/bootstrapserver"
+	"storj.io/storj/pkg/dht"
```

Comment:
we probably should be getting rid of dht package, it's mainly indirection that doesn't help with clarity.

Reply:
We do use the dht package to create a testing interface for the routing table...

Reply:
Not really, I mean we do use it in tests however I don't know a good reason to keep it in `dht` and keep referring to it. Alternatively the `RoutingTable` interface would make more sense as an interface in kademlia package, rather than `dht`.